packages/esm-patient-medications-app/src/add-drug-order/drug-order-form.component.tsx

```diff
@@ -167,6 +167,22 @@ export function DrugOrderForm({
     promptBeforeClosing(() => isDirty);
   }, [isDirty, promptBeforeClosing]);
 
+  // reset the dosage information if set to free text dosage
```

Comment:
```suggestion
  // reset dosage fields when toggling between free text and coded dosage
```

Nit: this edit might be a touch clearer.